Orange/data/io.py

```diff
@@ -807,7 +807,11 @@ class PickleReader(FileFormat):
 
     def read(self):
         with open(self.filename, 'rb') as f:
-            return pickle.load(f)
+            table = pickle.load(f)
+            if not isinstance(table, Table):
+                raise Exception("Not a table.")
```

Comment:
Don't raise instances of `Exception`; `Exception` is just the base class for (most) other exceptions. I guess that `TypeError` would be the appropriate type of exception for this.

You can't even catch `Exception` without pylint complaining (unless silenced).